content/events/consensus-depin-austin.md

```diff
@@ -0,0 +1,46 @@
+---
+eventName: "Consensus - Austin DePIN" #the name of your track or event and its mandatory
+
+eventDescription: "The most DePINtastic Consensus side event of them all, where Web3 and real world come together in an alchemical marriage." #short description of your track or event limiting to 100-150 characters
+
+#Topics will show up as labels on the event card.Only a maximum of 4 tags will be displayed on the event card. Some references for topics - Blockchain, Web3, Cryptocurrency, Tech Talks, Workshop, etc.
+
+eventTopic: 
+  - "DePIN"
+  - "<Topic_2>"
+  - "<Topic_3>"
+  - "<Topic_4>" 
+
+eventType: "Conference" #Type of the event: Please choose one among the below options or just leave it blank
+
+location: "Austin, United States" #Specify the location of the event.If you aren't sure about the location then mention "Location TBD"
+
+venueName: "The Perch ATX" #the event venue name (will show up on the event card) or just leave it blank
+
+venueAddress: "74 Trinity St, Austin, TX 78701" #the event venue address (will show up on a map) or just leave it blank
+
+venueMapsLink: "https://maps.app.goo.gl/74hSTrnR4oaKdzb39" #the event venue Map link (will show up on a map) or just leave it blank
+
+website: "https://lu.ma/DePINAustin" #make sure to have all the relevant information: dates, venue, program, ticketing (if any), etc. or just leave it blank
+
+startDate: "05/30/2024" # the start date of the event , date format is MM/DD/YYYY eg: if it is February 16th 2023 => 02/16/2023
+
+endDate: "05/30/2024" #the end date of the event , date format is MM/DD/YYYY eg: if it is February 18th 2023 => 02/18/2023
+
+tag: "Industry Event" #Mention any one of the event types. Don't leave it blank.
+
+dateTBD: false  #If you don't have concrete dates, add the approximate dates & set dateTBD: true.
+
+#Preferred means of contacts - this is a placeholder for email(for eg:  - email|mailto:<email_id>) and other social handles like Twitter, LinkedIn, Discord, etc. (for eg.   - 'twitter|https://twitter.com/IPFS/status/1629199396700098560?s=20')
+
+preferredContacts:
+  - '<contact_means>|<contact_url>'
+
+#Event host names and their respective logos(preferred size is 48px width, 48px height)-place the logo file on the path 'public/uploads' for eg.   - IPFS|ipfs-logo.png
+
+eventHosts:
+  - Impossible Cloud|<host_logo>
```

Comment:
@laeccles Please share logos for all the hosts and mention their names along with them to identify the same. Cc @pandeyanujk